templates/jitsi_meet_config.js.j2

```diff
@@ -305,13 +353,13 @@ var config = {
         enabled: true,
 
         // Use XEP-0215 to fetch STUN and TURN servers.
-        // useStunTurn: true,
+        useStunTurn: {{ jitsi_meet_use_stun_turn }},
 
         // The STUN servers that will be used in the peer to peer connections
         stunServers: [
```

Comment:
Maybe we should create a variable `jitsi_meet_stun_servers`

Reply:
You are right, even if you don't install your own, it will be necessary for those who want to use third-party servers.

I'm already working on that improvement.

Reply:
Done! `jitsi_meet_stun_servers` list added